spec/requests/api/v1/accounts_spec.rb

```diff
@@ -94,82 +137,93 @@
       end
 
       it 'changes reblogs option' do
-        post :follow, params: { id: other_account.id, reblogs: true }
+        post "/api/v1/accounts/#{other_account.id}/follow", headers: headers, params: { reblogs: true }
 
-        json = body_as_json
-
-        expect(json[:following]).to be true
-        expect(json[:showing_reblogs]).to be true
-        expect(json[:notifying]).to be false
+        expect(body_as_json).to include({
+          following: true,
+          showing_reblogs: true,
+          notifying: false,
+        })
       end
 
       it 'changes notify option' do
-        post :follow, params: { id: other_account.id, notify: true }
-
-        json = body_as_json
+        post "/api/v1/accounts/#{other_account.id}/follow", headers: headers, params: { notify: true }
 
-        expect(json[:following]).to be true
-        expect(json[:showing_reblogs]).to be false
-        expect(json[:notifying]).to be true
+        expect(body_as_json[:following]).to be true
+        expect(body_as_json[:showing_reblogs]).to be false
+        expect(body_as_json[:notifying]).to be true
       end
 
       it 'changes languages option' do
-        post :follow, params: { id: other_account.id, languages: %w(en es) }
+        post "/api/v1/accounts/#{other_account.id}/follow", headers: headers, params: { languages: %w(en es) }
 
-        json = body_as_json
-
-        expect(json[:following]).to be true
-        expect(json[:showing_reblogs]).to be false
-        expect(json[:notifying]).to be false
-        expect(json[:languages]).to match_array %w(en es)
+        expect(body_as_json[:following]).to be true
```

Comment:
Minor style note here is that there are a mix of repeated expect calls (like right here) and singular expect calls (line ~142 above). Not critical to this migration, but would be nice to keep consistent.